Editor/Utility/CinemachineEditorAnalytics.cs

```diff
@@ -0,0 +1,260 @@
+                    CollectData(vcamChildren[c], id + "." + c, ref vcamDatas);
+            }
+        }
+
+        [Serializable]
+        struct ProjectData
+        {
+            public int brain_count;
+            public int vcam_count;
+            public int cam_count;
+            public List<VcamData> vcams;
+            public float time_elapsed;
+        }
+
+        [Serializable]
+        struct VcamData
+        {
+            public string id;
+            public string vcam_class;
+            public bool has_follow_target;
+            public bool has_lookat_target;
+            public string blend_hint;
+            public bool inherit_position;
+            public string standby_update;
+            public string mode_overwrite;
+            public string body_component;
+            public string aim_component;
+            public string noise_component;
+            public int custom_component_count;
+            public string[] extensions;
+            public int custom_extension_count;
+
+            public VcamData(string id, CinemachineVirtualCameraBase vcamBase)
+            {
+                this.id = id;
+                vcam_class = GetTypeName(vcamBase.GetType());
+                has_follow_target = vcamBase.Follow != null;
+                has_lookat_target = vcamBase.LookAt != null;
+                blend_hint = "";
+                inherit_position = false;
+                standby_update = vcamBase.m_StandbyUpdate.ToString();
+                mode_overwrite = "";
+                body_component = "";
+                aim_component = "";
+                noise_component = "";
+                custom_component_count = 0;
+                custom_extension_count = 0;
+                extensions = Array.Empty<string>();
+            }
+        }
+        
+        static void GetExtensions(CinemachineVirtualCameraBase vcamBase, 
+            out string[] vcamExtensions, out int customCount)
+        {
+            customCount = 0;
+
+            // collect extensions on vcam
+            var extensions = vcamBase.mExtensions;
+            if (extensions != null)
+            {
+                vcamExtensions = new string[extensions.Count];
+                for (var i = 0; i < extensions.Count; i++)
+                {
+                    vcamExtensions[i] = (GetTypeName(extensions[i].GetType(), ref customCount));
+                }
+            }
+            else
+            {
+                vcamExtensions = Array.Empty<string>();
+            }
+        }
+
+        static void GetComponents(CinemachineComponentBase[] cmComps,
+            out string body, out string aim, out string noise, out int customCount)
+        {
+            customCount = 0;
+            body = aim = noise = "";
+            if (cmComps != null)
+            {
+                for (var i = 0; i < cmComps.Length; i++)
+                {
+#if CINEMACHINE_EXPERIMENTAL_VCAM
+                    if (cmComps[i] == null) continue;
+#endif
```

Comment:
cmComps[i] can be null only with newVirtualCamera. Though, this check does not hurt in either case.